### PR TITLE
Add core files.

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# credentials-context
\ No newline at end of file
+# credentials-context
+
+See: https://github.com/w3c/vc-data-model
```
Comment (davidlehn) [inline README.md:3]: You could slurp in and adjust the did-context README: https://github.com/digitalbazaar/did-context/blob/master/README.md

Comment (mattcollier) [reply README.md:3]: @davidlehn i'll add an issue for improving the readme with your suggestion.  I don't want this to hold up a 1.0 release.

**Review by davidlehn — APPROVED** [review]
If you need an immediate release ok.  Please note in an issue:
- Update README.md (perhaps adapt from did-context one)
- Add LICENSE[.md] file, refer to it from package.json, and perhaps use similar BSD/W3C note in README that did-context has.

**Comment by mattcollier** [comment]
@davidlehn thank you!  License file added, issue created for updating the readme.

**Pull request merged**